frontend/src/components/GroupView/index.tsx

```diff
@@ -1,11 +1,5 @@
 import React, { ReactElement, useState, useEffect } from 'react';
-import type {
-  Group,
-  SamwiseUserProfile,
-  Task,
-  GroupTaskMetadata,
-  SubTask,
-} from 'common/types/store-types';
+import type { Group, SamwiseUserProfile, Task, GroupTaskMetadata } from 'common/types/store-types';
 import type { FirestoreUserData } from 'common/types/firestore-types';
 import MiddleBar from './MiddleBar';
 import RightView from './RightView';
```

Comment:
merge in master into this branch so that these changes from #672 won't reappear here.